conda/lock.py

```diff
@@ -20,13 +20,31 @@
 import logging
 import os
 import time
+from glob import glob
+from os.path import abspath, isdir, join
 
+from conda.install import rm_rf
+from .compat import range
 from .exceptions import LockError
 
 LOCKFN = '.conda_lock'
 
+# Keep the string "LOCKERROR" in this string so that external
+# programs can look for it.
+LOCKSTR = """\
+LOCKERROR: It looks like conda is already doing something.
+The lock %s was found. Wait for it to finish before continuing.
+If you are sure that conda is not running, remove it and try again.
+You can also use: $ conda clean --lock
+"""
 
 stdoutlog = logging.getLogger('stdoutlog')
+log = logging.getLogger(__name__)
```

Comment:
I probably should have changed the `logging` import.  This line *should* just be `log = getLogger(__name__)` and above should be `from logging import getLogger`.